src/Submissions/SubmissionsApi.php

```diff
@@ -57,21 +59,20 @@ public function getSubmission($bucketName, $submissionUid)
     {
         $requestData = $this->getDefaultRequestData('query', []);
         $requestUri = vsprintf('buckets/%s/submissions/%s', [$bucketName, $submissionUid]);
-        $result =  $this->sendRequest($requestUri, $requestData, self::HTTP_METHOD_GET);
-        return true === $result ? [] : $result;
+        return $this->sendRequest($requestUri, $requestData, self::HTTP_METHOD_GET);
     }
 
     /**
      * @param string $bucketName
-     * @param string $submissionId
+     * @param string $submissionUid
      * @param array $params
```

Comment:
Remove this line as we don't use `$params` array anymore (`createSubmission` and `updateSubmission` methods).